stable/metricbeat/templates/deployment.yaml

```diff
@@ -0,0 +1,75 @@
+# Deploy singleton instance in the whole cluster for some unique data sources, like kube-state-metrics
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ template "metricbeat.fullname" . }}
+  labels:
+    app: {{ template "metricbeat.name" . }}
+    chart: {{ template "metricbeat.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  template:
+    metadata:
+      labels:
+        app: {{ template "metricbeat.name" . }}
+        release: {{ .Release.Name }}
+      annotations:
+        checksum/modules: {{ toYaml .Values.deployment.modules | sha256sum }}
+        checksum/config: {{ toYaml .Values.deployment.config | sha256sum }}
+{{- if .Values.deployment.podAnnotations }}
+  {{- range $key, $value := .Values.deployment.podAnnotations }}
+        {{ $key }}: {{ $value }}
+  {{- end }}
+{{- end }}
+    spec:
+      serviceAccountName: metricbeat
```

Comment:
use the template as defined in https://github.com/helm/helm/blob/master/docs/chart_best_practices/rbac.md

Reply:
I replaced this into the same one using at daemonset.yaml.